doc/topics/installation/osx.rst

```diff
@@ -29,26 +88,38 @@ Installation from Homebrew
 
 It should be noted that Homebrew explicitly discourages the `use of sudo`_:
 
-    Homebrew is designed to work without using sudo. You can decide to use it but we strongly recommend not to do so. If you have used sudo and run into a bug then it is likely to be the cause. Please don’t file a bug report unless you can reproduce it after reinstalling Homebrew from scratch without using sudo
+    Homebrew is designed to work without using sudo. You can decide to use it
+    but we strongly recommend not to do so. If you have used sudo and run into a
+    bug then it is likely to be the cause. Please don't file a bug report unless
+    you can reproduce it after reinstalling Homebrew from scratch without using
+    sudo
 
-.. _use of sudo: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/FAQ.md#sudo
+.. _use of sudo: https://docs.brew.sh/FAQ#why-does-homebrew-say-sudo-is-bad
 
 Installation from MacPorts
 ==========================
 
-Macports isolates its dependencies from the OS, and installs salt in /opt/local by default, with config files under /opt/local/etc/salt. For best results, add /opt/local/bin to your PATH.
+Macports isolates its dependencies from the OS, and installs Salt in
+``/opt/local`` by default, with config files under ``/opt/local/etc/salt``. For
+best results, add ``/opt/local/bin`` to your PATH.
 
 .. code-block:: bash
 
     sudo port install salt
 
-Variants allow selection of python version used to run salt, defaulting to python27, but also supporting python34, python35, and python36. To install salt with Python 3.6, use the python36 variant, for example:
+Variants allow selection of the python version used to run Salt, defaulting to
+python27, but also supporting python34, python35, and python36. To install Salt
```

Comment:
This is outdated information. We no longer support python27 or python34. Can you update what the default version is and remove python34?

Reply:
Done, though the ports package states that it supports python38 on macOS.
https://ports.macports.org/search/?q=salt